README.md

```diff
@@ -108,3 +108,27 @@ In the main function window, click on `+ Add trigger`. Search and select _S3_ in
 <kbd><img src="./images/trigger.png"></kbd>
 
 :tada: You're ready to start sending custom events and purchases to Braze!
+
+
+### Monitoring and Troubleshooting
+
+The function logs helpful messages while it is executing to help you understand if is processing the file correctly. The logs are available in the CloudWatch service. Under the **Monitor** tab, click on `View logs in CloudWatch`, and select the appropriate function log stream.
```

Comment:
During execution, the function will log messages to help you understand if the file is being processed correctly. These logs are available on the CloudWatch service.